metrics/internal/collectors/storageconsumer.go

```diff
@@ -60,6 +61,12 @@ func NewStorageConsumerCollector(opts *options.Options) *StorageConsumerCollecto
 			[]string{"storage_consumer_name"},
 			nil,
 		),
+		StorageQuotaUtilizationRatio: prometheus.NewDesc(
+			prometheus.BuildFQName("ocs", "storage_client", "storage_quota_utilization_ratio"),
+			`StorageQuotaUtilizationRatio of ODF Storage Client`,
+			[]string{"odf_client_name"},
```

Comment:
from line#151, we understand that the `odf_client_name` label is populated by concatenating client_name and cluster_name. Question is why not we add two individual labels (something like "client_name" and "client_cluster_name") and populate them separately? (a multiple label example can be seen here with `StorageConsumerMetadata` description)

Reply:
There is no specific reason for using multiple labels instead of using a single label. Do you suggest not to use concat?

Reply:
No, nothing restricting us to add one single label concatenating multiple values (from other labels), if you prefer that way it is perfectly fine. Raised this question, because we have the option to add multiple labels individually and their combination determines the uniqueness (so why restrict our self by using a single label) (=

Another plus point (of using multiple labels) is about ease of understanding. For example,
 if you have three labels like `namespace`, `cluster-name` and `host-name`, it is much easier to understand the metric object with these three distinct labels, rather than adding a single label with all the values concatenated.

Above example with separate labels
```
Name                  namespace            host             clustername                 Value
metric-abc             my-ns           10.9.31.109         my_cluster_1                    1
```

Same example with concatenated metric label
```
Name                      my_unique_label                         Value
metric-abc          my_cluster_1_my-ns_10.9.31.109                  1
```

As a general thumb-rule, prefer the concatenated values if we are creating a new label by combining some internal values (which is not exposed to the user/admin), but try to create separate labels if the given labels are known ones (like namespace, cluster-name etc).